### DISCUSSION TITLE
Windows like Snap Assist

### DISCUSSION THREAD

**Discussion by asimcard** [rxhanson/RectanglePro-Community#133] [Ideas]
On Windows if you snap a Window the open space that is unsnapped will show the remaining windows on that desktop that you can snap on as well. Here is an image that shows this behavior. Is this possible in the current release?

![image](https://user-images.githubusercontent.com/25911367/194375832-ae44febc-6cb3-4fcd-b9ad-76475491187f.png)

**Comment by rxhanson** [comment]
This is not possible in the current release. It gets requested from time to time, but is typically lower priority than other items since it is a pretty complex feature to implement as a 3rd party.

**Reply by asimcard** [reply]
Ok thanks for the reply, to avoid making another thread I just bought the pro version since I use rectangle so much. One thing I noticed is there is no hide menu bar icon option. Is this by design?

**Reply by rxhanson** [reply]
That option is in the Menu & Icon tab. (Thanks for making the jump to pro!)
